blockchain/state_transition.go

```diff
@@ -283,7 +288,14 @@ func (st *StateTransition) TransitionDb() (ret []byte, usedGas uint64, kerr kerr
 			return nil, 0, kerr
 		}
 	}
-	st.refundGas()
+
+	if rules.IsKore {
```

Comment:
If next hard fork enabled like Mantle, this condition should be changed. It need to cover the next hard fork. Otherwise, this should have a precondition, if Mantle hard fork enabled, kore hard fork should be enabled too.

Reply:
lower hf versions are automatically set as true. That is, if Mantle hardfork is enabled, kore hard fork is enabled too.

Reply:
Oh, I see, thanks.